app/templates/buckets/new.html

```diff
@@ -4,9 +4,13 @@
 
 {% block main_column %}
 
-<form action="/buckets/create" method="post">
+{% if error and not error.error %}
+<div class="error">{{ error.message }}</div>
```

Comment:
I tend to indent these to make clear the `<div>` would be displayed if the `if` condition is true.
Again, doesn't look like a massive difference in this case but when you have more "complicated" templates can be very helpful.

(And yes, the rendered output would have an indentation which may seem odd but as indentation is ignore in HTML I would err on the side of clarity for us)

Reply:
Agreed